labs/litellm_service/request.py

```diff
@@ -27,31 +26,6 @@ class RequestLiteLLM:
     def __init__(self):
         self.api_key = settings.LITELLM_API_KEY
 
-    def completion(self, messages, model="llm-model"):
-        """
-        messages expected to be in the following format:
-        [
-            {
-                "role": "user",
-                "content": ""
-            }
-        ]
-        Where role can be "user", "assistant", "system".
-        And content is the body of the message.
-        """
-        headers = {
-            "Accept": "application/json",
-            "API-Key": self.api_key,
-        }
-        data = {"messages": messages}
-        result = requests.post(
-            f"http://0.0.0.0:4000/chat/completions?model={model}",
-            headers=headers,
-            json=data,
-        )
-
-        return result.json()
-
```

Comment:
Why was this removed?

Reply:
This was deprecated and no one had removed it yet